src/ansys/fluent/core/workflow.py

```diff
@@ -333,6 +333,9 @@ def _tasks_with_matching_attributes(self, attr: str, other_attr: str) -> list:
                 matches.append(task)
         return matches
 
+    def display_name(self):
+        return self.get_id()
```

Comment:
Something might be awry here. E.g., Display name would be "Import Geometry" for id "TaskObject1".